tmt/scripts/install-podman.sh

```
@@ -0,0 +1,39 @@
+COMPOSE_VERSION="fc$(echo "$COMPOSE" | cut -d'-' -f2)"
+CUSTOM_PODMAN_URL="https://kojipkgs.fedoraproject.org//packages/podman/${PODMAN_VERSION}/1.${COMPOSE_VERSION}/${ARCH}/podman-${PODMAN_VERSION}-1.${COMPOSE_VERSION}.${ARCH}.rpm"
+
+# Install Podman based on the requested version:
+#   - "nightly": latest nightly build from rhcontainerbot/podman-next COPR repository
+#   - "latest": latest stable release from official Fedora repositories
+#   - other: install the exact RPM from Fedora Koji
+if [[ "$PODMAN_VERSION" == "nightly" ]]; then
+    sudo dnf copr enable -y rhcontainerbot/podman-next
+    sudo dnf install -y podman --disablerepo=testing-farm-tag-repository 
+    PODMAN_VERSION="$(dnf --quiet \
+        --repofrompath=podman-next,https://download.copr.fedorainfracloud.org/results/rhcontainerbot/podman-next/fedora-$(rpm -E %fedora)/${ARCH}/ \
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)